src/components/WorkOrder/TasksAndSors/TasksAndSorsView.js

```diff
@@ -9,23 +10,21 @@ const TasksAndSorsView = ({ tabName, tasksAndSors }) => {
     return !tasks.every(originalQuantityIsEqualToCurrentQuantity)
   }
```

Comment:
Probably also worth extracting this one (maybe into a `tasks.js` util file) and getting some unit tests around it